package.json

```diff
@@ -1,18 +1,24 @@
 {
-  "name": "virtual-labs-experiment-generator",
+  "name": "virtual-labs-experiment-authoring-environment",
   "displayName": "Virtual Labs Experiment Authoring Environment",
   "description": "A VS Code Plugin to handle all the common tasks invovled in virtual labs experiment development.",
   "version": "0.0.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/virtual-labs/tool-vscode-plugin/tree/testing"
```

Comment:
You do not need to give the repo link to the testing branch, just give a link to the root, i.e. main branch

Reply:
done